var/spack/repos/builtin/packages/wonton/package.py

```diff
@@ -49,15 +50,14 @@ class Wonton(CMakePackage):
     depends_on('mpi', when='+mpi')
     depends_on('flecsi', when='+flecsi')
 
+    depends_on('jali@1.1.6', when='wonton@3.2.1: +jali')
```

Comment:
There is no `wonton@3.2.1` that I know of - was this a typo (maybe `wonton@1.3.2:` was the intent)?

Also, I think it works fine to say `when=<this-package-name>...` but generally for constraints on "this" package, you can leave off the package name.

Reply:
Yes it was a typo thank you